QuickFixers.Data/DataBase/DatabaseSelections.cs

```diff
@@ -70,5 +70,190 @@ public static DataTable SelectUser(User loggingUser)
             }
             return returnResults;
         }
+
+        //Used to retrieve Service Provider info and save to Session for later use
+        public static DataTable SelectSP(int sessionUserID)
+        {
+            DataTable returnResults = new DataTable();
+            try
+            {
+                MySqlConnectionStringBuilder connectionStringBuilder = new MySqlConnectionStringBuilder();
+                connectionStringBuilder = DatabaseExtensions.ToConnectionStringBuilder(connectionStringBuilder);
+
+                using (var connectionDB = new MySqlConnection(connectionStringBuilder.ToString()))
+                {
+                    using (var sqlQuery = new MySqlCommand("quickFixers.SelSPUser", connectionDB))
+                    {
+                        sqlQuery.CommandType = CommandType.StoredProcedure;
+                        sqlQuery.Parameters.AddWithValue($"@UsrID", sessionUserID);
```

Comment:
Please update parameter name to `@UserID`

Reply:
Updated to UserID in App code and db Stored Procedure